packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -44,7 +48,27 @@ export const COMMON_ESBUILD_OPTIONS = {
 } as const;
 
 // build conditions used by esbuild, and when resolving custom `import` calls
-export const BUILD_CONDITIONS = ["workerd", "worker", "browser"];
+export function getBuildConditions() {
+	const envVar = getBuildConditionsFromEnv();
+	if (typeof envVar === "string") {
+		return envVar.split(",");
```

Comment:
can `envVar` here be anything? if not, who's going to error if unsupported values are passed? will that be esbuild?

Reply:
I "think" it could literally be anything. And yes, if it does fail then we would get some message from esbuild.
Given that this is for somewhat advanced private use for now, I am happy with that.

The only reason I added the check for the platform option was to narrow the types.

Reply:
totes fine by me. Was more FMI (for my info) kinda questions